src/runtime/manifest-parser.pegjs

```diff
@@ -1482,6 +1486,21 @@ PrimaryExpression
     return toAstNode<AstNode.TextNode>({kind: 'text-node', value: txt.join('')});
   }
 
+Units = whiteSpace? name: Unitname {
+  // TODO: Support complex units like metres per second.
+  return [name];
+}
+
+Unitname
+  = unit:('day'
+  / 'hour'
+  / 'minute'
+  / 'second'
+  / 'millisecond'
```

Comment:
do we really want to be millisecond granular on the manifest level?

Reply:
I was attempting to match Epoch time and the ttl code that I believe uses milliseconds as the base unit. I think we should keep the capability as these units can be used for other things than ttl and it might be helpful, but maybe we should default to seconds?